src/bmi_cfe.c

```diff
@@ -1484,6 +1486,11 @@ static int Initialize (Bmi *self, const char *file)
     cfe_bmi_data_ptr->gw_reservoir.coeff_secondary = 0.0;                // 0.0 means that secondary outlet is not applied
     cfe_bmi_data_ptr->gw_reservoir.exponent_secondary = 1.0;             // linear
 
+    // Not used in gw reservoir
```

Comment:
Just a comment, nothing to change. We should really move this and a lot of the code that directly precedes this into the `init_soil_reservoir` function that is called right after this. The initialization code is littered all over the place.